rocketpool-cli/wallet/purge-keys.go

```diff
@@ -0,0 +1,44 @@
+package wallet
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli"
+
+	"github.com/rocket-pool/smartnode/shared/services/rocketpool"
+	cliutils "github.com/rocket-pool/smartnode/shared/utils/cli"
+)
+
+func purgeKeys(c *cli.Context) error {
```

Comment:
Deleting the files isn't enough- the keys are held in memory by the running process. The `validator` container should be restarted on the success of `purgeKeys` (eth2 should be restarted if nimbus is selected).

Reply:
Very good call Patches. I'm worried about choosing a certain behavior for each client (ex. restart eth2 only for nimbus) as we would need to track any changes to clients that could have an impact on what needs to be restarted. I'm thinking we should play it safe and restart the whole service. What do you think @jclapis?

Reply:
Update: Now I'm calling the RestartValidator function which checks what needs to be restarted according to the client being used.